src/helpers/shared/user-login.js

```diff
@@ -76,3 +78,9 @@ export function getCostApi() {
     getResource: (path) => costApi.axios.get(`${path}?limit=20000`),
   };
 }
+
+export const getInventoryGroupsApi = (apiProps) => {
+  return {
+    getInventoryGroups: () => inventoryApi.axios.get(`${INVENTORY_API_BASE}/groups`),
+  };
+};
```

Comment:
We are not using `apiProps` in here, I don't think we need to consume them
```suggestion
export const getInventoryGroupsApi = () => inventoryApi.axios.get(`${INVENTORY_API_BASE}/groups`);
```